src/main/java/se/sundsvall/supportmanagement/api/model/notification/Notification.java

```diff
@@ -0,0 +1,256 @@
+package se.sundsvall.supportmanagement.api.model.notification;
+
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Null;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
+import se.sundsvall.supportmanagement.api.validation.groups.OnCreate;
+import se.sundsvall.supportmanagement.api.validation.groups.OnUpdate;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public class Notification {
+
+	@Null(groups = {OnCreate.class, OnUpdate.class})
+	@Schema(description = "Unique identifier for the notification", example = "123e4567-e89b-12d3-a456-426614174000")
+	private String id;
+
+	@DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+	@Schema(description = "Timestamp when the notification was created", example = "2000-10-31T01:30:00.000+02:00")
+	private OffsetDateTime created;
+
+	@DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+	@Schema(description = "Timestamp when the notification was last modified", example = "2000-10-31T01:30:00.000+02:00")
+	private OffsetDateTime modified;
+
+	@NotBlank
+	@Schema(description = "Owner of the notification", example = "Test Testorsson")
+	private String owner;
```

Comment:
ownerName in order to describe what part of the owner we are expecting here (id, full-name, firstname, userid, etc)? 

(We have ownerId, so according to that pattern this attribute should be: ownerName or ownerFullName?)